docs/for-ops/security-best-practices.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: security-best-practices
+title: Security Best Practices
+sidebar_label: Security Best Practices
+---
+
+The Akamai App Platform comes with many security capabilities. It is up to the platform administrator to make sure these capabilities are enabled and correctly used. The following checklist can be used to make sure security is implemented to follow best-practices when running App PLatform on LKE.
```

Comment:
It is up to the platform administrator => It is platform administrator responsibility